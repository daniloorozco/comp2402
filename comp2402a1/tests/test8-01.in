loser
loser
hey
hey
hey
well
well
well
open
why
loser
loser
loser
