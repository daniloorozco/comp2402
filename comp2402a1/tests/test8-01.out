loser
hey
well
open
why
